connectors/graphql/src/main/java/io/camunda/connector/graphql/model/GraphQLRequestWrapper.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.graphql.model;
+
+import io.camunda.connector.common.auth.Authentication;
+import java.util.Objects;
+import javax.validation.constraints.NotBlank;
+
+public class GraphQLRequestWrapper {
+  @NotBlank private GraphQLRequest graphql;
+  private Authentication authentication;
```

Comment:
Shouldn't it be marked as `@Secret`?

Reply:
Not necessarily, we do not want to replace any secrets in these two attributes. This is just a helper class to handle the graphql namespace. (graphql.url, graphql.method, etc.)